setup.py

```diff
@@ -2,33 +2,40 @@
 
 from setuptools import find_packages, setup
 
-VERSION = '0.0.3'
-PYTHON_REQUIRES = '3.8'
+VERSION = "0.0.3"
+PYTHON_REQUIRES = "3.8"
```

Comment:
why 3.8, should we bump it up?

Reply:
I'll create separate ticket for it, i was focusing solely on development workflow here